[WC-183] Update changelogs of tooling

IIsaku opened the pull request:
## This PR contains
- [ ] Bug fix
- [ ] Feature
- [ ] Refactor
- [ ] Documentation
- [x] Other (describe)

## What is the purpose of this PR?
Update changelogs due to release.

## Relevant changes
- Moved changes to a specific release for `generator-widget` & `pluggable-widgets-tools`.
- Add forgotten notes for PIW tools changes introduced in #593.

## What should be covered while testing?
Nothing.

Keraito commented on packages/tools/generator-widget/CHANGELOG.md:
```diff
@@ -1,12 +1,15 @@
-- Set required Node.js version to v12 or higher for the generator itself and the generated widget projects, since Node.js v10 is EOL at the end of April 2021 and pluggable widgets tools requires Node.js v12.
 
-## [major.minor.patch] - YYYY-MM-DD
+-   Fixes failing `npm install` executions when scaffolding a widget project with the widget generator with NPM v7.x.x.
+-   Explains users in the README of generated widget project how to install dependencies with NPM v7.x.x.
+-   Set required Node.js version to v12 or higher for the generator itself and the generated widget projects, since Node.js v10 is EOL at the end of April 2021 and pluggable widgets tools requires Node.js v12.
```
It's fine since this PR is small, but these random whitespace changes are really annoying during review 😭 For future PRs, please only include them if they have added value.

IIsaku replied:
I will make sure to do these changes in a separate commit or separate PR next time depending on the situation.

mmehmetAliIzci commented on packages/tools/generator-widget/CHANGELOG.md:
```diff
@@ -1,12 +1,15 @@
-- Set required Node.js version to v12 or higher for the generator itself and the generated widget projects, since Node.js v10 is EOL at the end of April 2021 and pluggable widgets tools requires Node.js v12.
 
-## [major.minor.patch] - YYYY-MM-DD
+-   Fixes failing `npm install` executions when scaffolding a widget project with the widget generator with NPM v7.x.x.
```
We should align all the comments in the same grammer IMO. Either

- **Fix** failing .....
- **Explain** ...

Or

- **Sets** required Node.js

IIsaku replied:
Changed it to your first proposal 🙂  . This grammar style is also used by https://keepachangelog.com/en/1.0.0/

IIsaku committed:
chore: update changelogs tools

- Add forgotten notes for another change to PIW tools.
- Move changes to different sections.

Keraito commented on packages/tools/generator-widget/CHANGELOG.md:
```diff
@@ -1,12 +1,18 @@
+
+### Changed
+
+- Explain users in the README of generated widget project how to install dependencies with NPM v7.x.x.
```
```suggestion
- Explain to users in the README of generated widget project how to install dependencies with NPM v7.x.x.
```

Keraito commented on packages/tools/generator-widget/CHANGELOG.md:
```diff
@@ -1,12 +1,18 @@
 
-## [major.minor.patch] - YYYY-MM-DD
+- Fix failing `npm install` executions when scaffolding a widget project with the widget generator with NPM v7.x.x.
+- Set required Node.js version to v12 or higher for the generator itself and the generated widget projects, since Node.js v10 is EOL at the end of April 2021 and pluggable widgets tools requires Node.js v12.
```
Not entirely whether we need to be so specific about _why_ we made the changes. It doesn't hurt, but not sure about the added value of it. No need to necessarily change it tho.

IIsaku replied:
I can understand your doubts. Here I think it adds value, because it clarifies this isn't a breaking change.

Keraito reviewed (dismissed):
Looks fine, 1 grammar change and another minor remark but that one is not blocking.

IIsaku committed:
chore(generator-widget): review: improve sentence

Co-authored-by: Chak Shun Yu <chakshunyu@gmail.com>
